docs/source/pillars/supporting.md

```diff
@@ -0,0 +1,217 @@
+
+```{list-table}
+:header-rows: 1
+:name: tab-business-continuity-subject
+
+* - Statement
+  - Guidance
+  - Importance
+* - You have a business continuity plan that includes consideration of loss of service for deployed TREs
+  - This may be due to downtime from service providers, a breach, or loss of power. Your plan should detail your process for managing loss of service for deployed TREs, and evaluation of impact of such loss
+  - Recommended
+* - You regularly test the aspects of your business continuity plan concerning TREs, and have a process in place to iterate the plan if required
+  -
+  - Recommended
+```
+
+## Project and programme management
+
+_What the organisation does to ensure effective management of programmes and projects._
+
+```{list-table}
+:header-rows: 1
+:name: tab-project-programme-management
+
+* - Statement
+  - Guidance
+  - Importance
+* - All projects requiring TRE are assigned project management support
+  - This may include budget management, tracking TRE status, managing communications with the TRE operations team, and other project support tasks
+  - Recommended
+* - Project management support assigned to a project requiring a TRE should not have direct access to the TRE
+  - Doing so ensures a separation between those able to access sensitive data, and those overseeing access to sensitive data
+  - Optional
+```
+
+## Knowledge management
+
+_What the organisation does to acquire, enrich, share, store, publish and enhance the expertise across the whole organisation._
+
+```{list-table}
+:header-rows: 1
+:name: tab-knowledge-management
+
+* - Statement
+  - Guidance
+  - Importance
+* - All features of your TRE infrastructure requiring input or attention from any stakeholders must be well documented
+  - This includes ensuring all documentation is discoverable, clear, and able to be easily updated based on stakeholder feedback
+  - Mandatory
+* - You should have an education programme in place to upskill stakeholders in the use and management of your TRE infrastructure
+  - This may include learning modules, workshops and other resources on how to effectively access and use a TRE, FAQs pages, and accessible pathways for additional support
+  - Recommended
+* - You should periodically carry out a training needs analysis (TNA) for all stakeholders included within your TRE provision
+  - At least once every 12 months you should assess the training needs of your stakeholders, and ensure they have easy access to all required training materials
+  - Recommended
+```
+
+## Financial management
+
+_All activities aimed at the efficient and effective management of money (funds) in such a manner as to allow the organisation to accomplish its objectives._
+
+```{list-table}
+:header-rows: 1
+:name: tab-financial-management
+
+* - Statement
+  - Guidance
+  - Importance
+* - All projects requiring a TRE must know and accept the financial cost of doing so
```

Comment:
Isn't this a requirement/capability of the user/researcher, rather than the TRE? Should this actually be `All TREs must know and accept the financial cost of doing so`?

Reply:
> Isn't this a requirement/capability of the user/researcher, rather than the TRE?

Are you saying it should be a requirement of the user/researcher, rather than TRE? If so I read what it already says as that, and your rewording as the opposite (it would then be a requirement of TRE teams

Reply:
@manics see my suggestion below (not sure why it didn't show up as a reply here)